server/pulp/plugins/importer.py

```diff
@@ -32,7 +32,7 @@ def get_downloader(config, url, **options):
         :raise ValueError: when the URL scheme is not supported.
         """
         url = urlparse(url)
-        nectar_config = importer_config_to_nectar_config(config.flatten())
+        nectar_config = importer_config_to_nectar_config(config.flatten(), working_dir='/tmp')
```

Comment:
I had thought the working_dir would not be specified here, and instead the streamer's use of importers specifically would set `working_dir=/tmp`